src/_P039_Thermosensors.ino

```diff
@@ -715,7 +899,7 @@ float readMax31856(struct EventStruct *event)
       if((log.reserve(66u))) {
         log = F("P039 : MAX31856 :");
 
-        for (int i = 1; i < MAX31856_NO_REG; ++i) {
+        for (uint8_t i = 1; i < MAX31856_NO_REG; ++i) {
```

Comment:
Is this correct? 
It always triggers me when a for loop doesn't start at 0. May be correct, but perhaps better to add a comment to why it does not start at 0?

Reply:
If it would be a feature i would comment - but it rather looks like a bug as well to me. Good find ! - > solved in next push